app/controllers/pledge_controller.rb

```diff
@@ -0,0 +1,40 @@
+class PledgeController < ApplicationController
```

Comment:
rename file to `pledges_controller.rb`; change the class name here to `PledgesController` with an S